README.md

```diff
@@ -137,6 +137,7 @@ A sample configuration file with all options is available [here](https://github.
     * `"check-lowercase"` enforces the rule that the first non-whitespace character of a comment must be lowercase, if applicable
     * `"check-uppercase"` enforces the rule that the first non-whitespace character of a comment must be uppercase, if applicable
 * `curly` enforces braces for `if`/`for`/`do`/`while` statements.
+* `enforce-trailing-comma` enforces trailing comma within multiline object literals.
```

Comment:
Rule can just be named "trailing-comma" (this will obviously change things in a few other places as well)